get-webpack-config.js

```diff
@@ -136,7 +136,7 @@ module.exports = function getWebpackConfig({
             clientLogLevel: isDebug ? 'info' : 'warning',
             contentBase: path.resolve(__dirname, 'dist'),
             hot: true,
-            open: true,
+            open: !isDebug,
```

Comment:
I don't recommend to make this change.

currently isDebug is only used for debugging console output - please revert it. 

If you would like this change - please send separate PR, so we can discuss it.

Reply:
fixed